code/components/gta-net-five/src/CloneExperiments.cpp

```diff
@@ -5040,6 +5040,40 @@ static HookFunction hookFunctionDiag([]
 	{
 		PedPoolDiagError();
 	});
-#endif
+#endif
+
+	fx::ScriptEngine::RegisterNativeHandler("GET_ENTITY_FROM_STATE_BAG_NAME", [](fx::ScriptContext& context)
+	{
+		int entityId = 0;
+		std::string bagName = context.CheckArgument<const char*>(0);
+
+
+		if (bagName.find("entity:") != std::string::npos)
+		{
+			bagName.replace(0, 7, "");
+			rage::netObject* obj = rage::netObjectMgr::GetInstance()->GetNetworkObject(atoi(bagName.c_str()), true);
+			if (obj)
+			{
+				int guid = getScriptGuidForEntity((fwEntity*)obj->GetGameObject());
```

Comment:
GetGameObject can be null and this'd potentially crash if that's the case. Needs a null check.